cockatrice/src/tab_game.cpp

```diff
@@ -620,11 +620,23 @@ void TabGame::actGameInfo()
 
 void TabGame::actConcede()
 {
-    if (QMessageBox::question(this, tr("Concede"), tr("Are you sure you want to concede this game?"),
-                              QMessageBox::Yes | QMessageBox::No, QMessageBox::No) != QMessageBox::Yes)
+    Player *player = players[localPlayerId];
```

Comment:
That's actually how I first wrote it, but apparently there is no QMap::get

http://doc.qt.io/qt-5/qmap-members.html

Reply:
[QMap::value()](http://doc.qt.io/qt-5/qmap.html#value)
The problem with operator[] is that if a key doesn't exists, it will create an empty new object.